packages/massa-web3/src/experimental/client/publicAPI.ts

```diff
@@ -93,8 +93,8 @@ export class PublicAPI {
         target_function: params.func,
         parameter: params.parameter ? Array.from(params.parameter) : [],
         caller_address: params.callerAddress,
-        coins: params.coins ? Mas.toString(params.coins, true) : null,
-        fee: params.fee ? Mas.toString(params.fee, true) : null,
+        coins: params.coins ? Mas.toString(params.coins) : null,
+        fee: params.fee ? Mas.toString(params.fee) : null,
```

Comment:
You've implemented something that is finally not used?

Reply:
This differs from what I've been told, but it seems to work like this.